src/migrations/20190813125249-create-article.js

```diff
@@ -10,19 +10,23 @@ module.exports = {
       slug: {
         type: Sequelize.STRING,
         allowNull: false,
-        unique: true,
+        unique: true
       },
       title: {
         type: Sequelize.STRING,
-        allowNull: false,
+        allowNull: false
       },
       description: {
         type: Sequelize.TEXT,
-        allowNull: false,
+        allowNull: false
+      },
+      taglist: {
```

Comment:
I do think tag list has to be an array, not a string

Reply:
It is now an array